### DISCUSSION TITLE
Realsense Camera to be Discontinued

### DISCUSSION THREAD

**Discussion by brightsnowzhao** [prl-mushr/mushr#56] [Hardware]
Intel will discontinue realsense camera early next year.  Now there's a two-month lead time to order (likely the last batch).  Are there alternative cameras we can use?

**Comment by palanc** [comment]
Hi @brightsnowzhao - Thanks for pointing this out! In the short term, we have added links in our BOM that currently have compatible Realsense cameras in stock. Here they are:

D435i: [https://www.digikey.com/en/products/detail/intel-realsense/82635D435IDKMP/14301643](https://www.digikey.com/en/products/detail/intel-realsense/82635D435IDKMP/14301643)

D435 (no imu): [https://www.digikey.com/en/products/detail/intel-realsense/82635AWGDVKPMP/9926003](https://www.digikey.com/en/products/detail/intel-realsense/82635AWGDVKPMP/9926003)

D455: [https://www.digikey.com/en/products/detail/intel-realsense/82635DSD455MP/12429609](https://www.digikey.com/en/products/detail/intel-realsense/82635DSD455MP/12429609)

In the longer term, you're right that Intel is discontinuing the T265 early next year, but how long they will continue to manufacture and sell stereo IR cameras (D435, D435i, and D455) is somewhat murky. After making the announcement that they would be 'winding down' the Realsense component of their business, they clarified that they would continue to support their stereo cameras. Particularly, several employees from Realsense have commented here: [https://github.com/IntelRealSense/librealsense/issues/9648](https://github.com/IntelRealSense/librealsense/issues/9648)

I'd also point out that although the lead-time on Intel's own online store could be related to this, it could also be a result of supply chain issues that are causing shortages throughout the electronics sector.

Over the next few months we will continue to evaluate the availability of Realsense cameras and explore the use of alternatives incase Intel does end up discontinuing its stereo cams.

**Reply by brightsnowzhao** [reply]
Thank you for your prompt feedback!
